jobs/mobile-developer.md

```diff
@@ -0,0 +1,57 @@
+# Mobile Developer (Android/iOS)
+
+We are looking for experienced and initiative Mobile Developers to grow our team and develop meaningful products together. We value openness and knowledge sharing, believe that communication is a key and develop the remote culture for around 6 years. We would like to work with someone who shares our values, acts independently and proactively.
+
+
+## How the job looks
+
+- We have jobs both for Android and iOS developers, experienced (or interested to learn) in React Native. We primarily use Kotlin for Android and Swift for iOS, yet certain Java or Objective-C knowledge might be useful.
+- You will work with a team of developers (3-5 people) on the current project developing new features.
+- Usually our projects last for about 1.5-2 years (but it could be more/less). Our developers don't work on more than 1 project at a time.
+- Salary 𝚫: 24k—48k USD.
+- For development we usually use Kanban or similar process in Trello, Pivotal of Jira, communicate in Slack channels and via Zoom calls, and store documentation in Notion. We also use [Holacracy](https://en.wikipedia.org/wiki/Holacracy) for organizational governance.
+- Apart from working on projects we will encourage you to gain and share knowledge, discuss your personal development and contribute to company's activities.
+- You'll have colleagues all around the globe, variety of channels in Slack like #animal-therapy, #cooking, #ihateremotework and remote team spirit.
+
+## How a good candidate looks
+
+- You have solid experience either in Android or iOS.
+- You wish to work with React Native, even if you have not much experience in it.
+- You have an opinion on how to architect the app and can compare MVP with MVVM.
+- You know how different components of mobile OSes work: push notifications, keystore & keychain, background tasks, GPS api, etc.
```

Comment:
@vshkl, I've added some of the OS APIs but maybe you have better opinion on what we should list here based on what APIs you've been working recently?

Reply:
Those are fine. I'd add `db`, probably, but it doesn't matter much, 'cuz we can't list all of them anyway.

Reply:
Good point 👍. Added.